aries-site/src/pages/templates/filtering.mdx

```diff
@@ -0,0 +1,81 @@
+import { Example } from '../../layouts';
+import {
+  FilteringCards,
+  FilteringLists,
+  FilteringWithRangeInput,
+  FilteringWithSelect,
+} from '../../examples';
+
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/filtering/FilteringCards.js"
+  showResponsiveControls={['laptop', 'mobile']}
+  template
+>
+  <FilteringLists />
+</Example>
+
+## What is filtering?
```

Comment:
```suggestion
## What Makes Up Filtering?
```

Reply:
I like this heading name but wondering again about sentence vs title case for capitalization.

Reply:
Let's go sentence case